From 8e158c16cd15fff08b42ed86edc55816d3387837 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Fri, 19 Feb 2016 16:04:24 +0200
Subject: [PATCH] configure.ac: fix paths to introspection tools

They need to be prefixed with $PKG_CONFIG_SYSROOT_DIR

Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
---
 configure.ac | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/configure.ac b/configure.ac
index 6d2b747..3b28463 100644
--- a/configure.ac
+++ b/configure.ac
@@ -706,9 +706,9 @@ if test "x$found_introspection" = "xyes"; then
    dnl python where to find libgoffice, so you might also need to set
    dnl LD_LIBRARY_PATH.
 
-   INTROSPECTION_SCANNER=`$PKG_CONFIG --variable=g_ir_scanner gobject-introspection-1.0`
-   INTROSPECTION_COMPILER=`$PKG_CONFIG --variable=g_ir_compiler gobject-introspection-1.0`
-   INTROSPECTION_GENERATE=`$PKG_CONFIG --variable=g_ir_generate gobject-introspection-1.0`
+   INTROSPECTION_SCANNER=$PKG_CONFIG_SYSROOT_DIR`$PKG_CONFIG --variable=g_ir_scanner gobject-introspection-1.0`
+   INTROSPECTION_COMPILER=$PKG_CONFIG_SYSROOT_DIR`$PKG_CONFIG --variable=g_ir_compiler gobject-introspection-1.0`
+   INTROSPECTION_GENERATE=$PKG_CONFIG_SYSROOT_DIR`$PKG_CONFIG --variable=g_ir_generate gobject-introspection-1.0`
    if test "x$INTROSPECTION_GIRDIR" = x; then
      INTROSPECTION_GIRDIR=`$PKG_CONFIG --variable=girdir gobject-introspection-1.0`
    fi
@@ -717,7 +717,7 @@ if test "x$found_introspection" = "xyes"; then
    fi
    INTROSPECTION_CFLAGS=`$PKG_CONFIG --cflags gobject-introspection-1.0`
    INTROSPECTION_LIBS=`$PKG_CONFIG --libs gobject-introspection-1.0`
-   INTROSPECTION_MAKEFILE=`$PKG_CONFIG --variable=datadir gobject-introspection-1.0`/gobject-introspection-1.0/Makefile.introspection
+   INTROSPECTION_MAKEFILE=$PKG_CONFIG_SYSROOT_DIR`$PKG_CONFIG --variable=datadir gobject-introspection-1.0`/gobject-introspection-1.0/Makefile.introspection
 else
    INTROSPECTION_SCANNER=
    INTROSPECTION_COMPILER=
-- 
2.7.0

